acceptance/bundle/telemetry/deploy-artifact-path-type/test.toml

```diff
@@ -13,3 +13,10 @@ Response.Body = '''
 [[Server]]
 Pattern = "PUT /api/2.0/fs/directories/Volumes/main/a/b/c/.internal"
 Response.StatusCode = 200
+
+[[Server]]
+Pattern = "GET /api/2.0/fs/directories/Volumes/main/a/b/c/.internal"
+Response.Body = '{}'
+# I'm adding 405 because that's what this test originally do. It's somewhat
+# surprising though that CLI can receive 405 and that does not result in error anywhere.
+Response.StatusCode = 405
```

Comment:
Was this for testing only?

Reply:
No, this change is required to make this test pass. Otherwise that call is made and since there is no handler, you now get t.Error().

Reply:
Got it -- so this still needs to be debugged in a follow-up.

@shreyas-goenka Could you figure out why this works when the backend returns a 405?